conf/distro/pelux.conf

```diff
@@ -6,6 +6,9 @@
 require conf/distro/poky.conf
 
 MAINTAINER = "Gordan Markuš <gordan.markus@pelagicore.com>"
+DISTRO = "pelux"
+DISTRO_NAME = "PELUX (Built on on Yocto 2.3)"
```

Comment:
Is it possible to get the Yocto version number from a variable so we don't need to think of updating it every half a year?